examples/README.md

```diff
@@ -0,0 +1,43 @@
+# XAITK-Saliency Examples
+This directory hosts the XAITK-Saliency examples.
+
+### 1. Requirements
+Most of the examples require [Jupyter Notebook](https://jupyter.org/) and [Pytorch](https://pytorch.org/).
+
+These can be installed manually using `pip` or with the following command:
+
+```bash
+poetry install -E example_deps
+```
+
+Some notebooks may require additional dependencies. Please see the first cell of each notebook on how to install the relevant packages.
+
+#### Run the notebooks from Colab
+
+Most of the notebooks have an "Open in Colab" button.
+Please right-click on the button, and select "Open Link in New Tab" to start a Colab page with the corresponding notebook content.
+
+To use GPU resources through Colab, please remember to change the runtime type to `GPU`:
+
+1. From the `Runtime` menu select `Change runtime type`
+1. Choose `GPU` from the drop-down menu
+1. Click `SAVE`
+This will reset the notebook and may ask you if you are a robot (these instructions assume you are not).
```

Comment:
> (these instructions assume you are not).

:rofl:

Reply:
I wish I could take credit for this, but I can't. I borrowed the skeleton of this README from the [MONAI tutorials](https://github.com/Project-MONAI/tutorials). I guess I should probably attribute this somewhere...